bitnami/solr/templates/ingress.yaml

```diff
@@ -9,7 +9,9 @@ metadata:
   {{- end }}
   annotations:
     {{- if .Values.ingress.certManager }}
+      {{- if not .Values.ingress.existingSecretName }}
     kubernetes.io/tls-acme: "true"
+      {{- end }}
```

Comment:
This seems to be weirdly indented, could you check it?

Reply:
the indentation here was intentional, as I personally find nested golang template logic to be easier to parse when indented. however, if it's standard practice to keep it at the same level as the content it's switching, I will update that.

Reply:
updated.